tests/outcome/test_outcome_resources.py

```diff
@@ -0,0 +1,333 @@
+import json
+from flask import url_for
+from urllib.parse import urlparse
+
+from dataservice.extensions import db
+from dataservice.api.common import id_service
+from dataservice.api.outcome.models import Outcome
+from dataservice.api.participant.models import Participant
+from dataservice.api.study.models import Study
+from tests.utils import FlaskTestCase
+
+OUTCOMES_URL = 'api.outcomes'
+OUTCOMES_LIST_URL = 'api.outcomes_list'
+
+
+class OutcomeTest(FlaskTestCase):
```

Comment:
We should merge #84 first and then you can probably remove a lot of these tests (bad input, missing req params) and put them in test_api.py.